### ISSUE TITLE
toRoundedUnit output type should be updated from int to float

### ISSUE THREAD

**Issue by JayLeoK** [bettercart/rescript-dinero#2]
Per the DineroJs docs, I think this output could also be a float: https://dinerojs.com/module-dinero#toRoundedUnit-examples

https://github.com/bettercart/rescript-dinero/blob/0156a210ab6a72bd56c40b195450d14016a94d17/src/Dinero.res#L89

**Comment by danwetherald** [comment]
Makes sense, I think the best way to handle this will be a second method for float:

`toRoundedFloatUnit`
`toRoundedIntUnit` this should not take an int and will be hard coded to 0

Thoughts?

**Comment by JayLeoK** [comment]
That seems like the best solution for either case. Thanks for quick responses. 👍
